### PR TITLE
Hue sensor support

### PR DESCRIPTION
**Pull request by mitchellrj**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22598

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/stream.markdown [hunk 1]
```diff
@@ -26,6 +26,47 @@ To enable this component, add the following lines to your `configuration.yaml` f
 stream:
 ```
 
+<p class='note'>
+Depending on your setup, you might need to set a base URL (`base_url`) inside the [http component](/components/http/) or in the parameters of this component.
```
Comment (robmarkcole) [inline source/_components/stream.markdown]: Should this be here?

Comment (klaasnicolaas) [reply source/_components/stream.markdown]: I rather think that all these changes in the stream markdown file do not even belong to this PR 🤔

**Comment by klaasnicolaas** [comment]
⚠️ Is it true that the changes to the stream component do not belong to this PR?

**Comment by mitchellrj** [comment]
Fixed sorry. You're right, a commit for another branch accidentally made its way into this PR.

File: source/_components/hue.markdown [hunk 2]
```diff
@@ -19,17 +19,18 @@ redirect_from:
...
 
 When you configure the Hue bridge from Home Assistant, it writes a token to a file in your Home Assistant [configuration directory](/docs/configuration/). That token authenticates the communication with the Hue bridge. This token uses the Address of the Hue Bridge. If the IP address for the Hue Bridge changes, you will need to register the Hue Bridge with Home Assistant again. To avoid this you may set up DHCP registration for your Hue Bridge, so that it always has the same IP address.
 
-Once registration is complete you should see the Hue lights listed as `light` entities. If you don't you may have to restart Home Assistant once more. Add these light entities to `configuration.yaml` file and restart Home Assistant once more to complete the installation.
+Once registration is complete you should see the Hue lights listed as `light` entities, Hue presence sensors listed as `binary_sensor` entites, and Hue temperature and light level sensors listed as `sensor` entities. If you don't you may have to restart Home Assistant once more.
```
Comment (klaasnicolaas) [inline source/_components/hue.markdown]:
```suggestion
Once registration is complete you should see the Hue lights listed as `light` entities, Hue presence sensors listed as `binary_sensor` entites and Hue temperature and light level sensors listed as `sensor` entities. If you don't you may have to restart Home Assistant once more.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
I think that we can merge it, after these suggested change 😉

**Comment by rohankapoorcom** [comment]
I think we need to pull this into `current` rather than `next` since this functionality was released in 0.92.

**Comment by klaasnicolaas** [comment]
We can cherry pick this into current

**Comment by klaasnicolaas** [comment]
Adjusting the base branch does not work. Because of this we get unrelated commits that go along with the merge. It is better to merge it in `next` and then perform a cherry pick to the `current` branch or you must perform a rebase to `current` yourself

**Review by rohankapoorcom — APPROVED** [review]
LGTM

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**